src/containers/blocks/DropBlock.js

```diff
@@ -0,0 +1,43 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {handleDragOver, handleDrop, handleDragLeave} from "../../functions/handleDragAndDrop"
+import {addBlockToTest} from "../../actions";
+
+class DropContainer extends Component {
+
+  handleDropCase(ev) {
+    const droppedItem = handleDrop(ev);
+    if(droppedItem !== null) {
+      this.props.addBlockToTest(droppedItem, ev.target);
+    } else {
+      console.log("Block couldn't be added properly.");
```

Comment:
If this console log is something that user should see it should be displayed as component instead, if not it's better to remove it

Reply:
User shouldn't see it, I've removed it.